### DISCUSSION TITLE
[Feature] index picture's path to use this information for searching

### DISCUSSION THREAD

**Discussion by francoisWeber** [immich-app/immich#12342] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi Immich Team, 
First of all: 👏🏼 super work for this server + app !

I've made some quick tests on my own instance and I think the feature i'm proposing here is missing.
What about **using the path's element of a picture/video as (hidden) attribute of it + index these elements in order to enhance the search experience** ? 
The context is the following: some people can spend time to sort, classify their picture into well structured directories - especially before apps like Immich appeared. I'm convinced in such cases, the full path of a photo can be super relevant to gain context about the picture, and so to increase a photo's "searchability" (recall increase). 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#6519

Note that since 1.113.0 there's now a folder browser view you can use.
